### PR TITLE
Cavity fault display has colored statuses and faulted codes are bolded

### PR DESCRIPTION
**Pull request by lopardog**

### CODE REVIEW

File: cavityFaultDisplay.py [hunk 1]
```diff
@@ -27,66 +27,87 @@ def __init__(self, parent=None, args=None, macros=None):
...
-            self.statusLabelCallback(statusLabel, fault)
+            self.statusLabelCallback(statusLabel, codeLabel, fault)
 
-            fault.pv.add_callback(partial(self.statusLabelCallback, statusLabel, fault))
-        verticalLayout.setSpacing(10)
+            fault.pv.add_callback(partial(self.statusLabelCallback, statusLabel, codeLabel, fault))
 
     @staticmethod
-    def statusLabelCallback(label: QLabel, fault: Fault, **kw):
+    def statusLabelCallback(label: QLabel, label2: QLabel, fault: Fault, **kw):
```
Comment (lisazacarias) [inline cavityFaultDisplay.py]: I would name these something slightly more descriptive! Like, what kind of labels are they?

**Review by lisazacarias — CHANGES_REQUESTED** [review]

File: cavityFaultDisplay.py [hunk 2]
```diff
@@ -27,66 +27,87 @@ def __init__(self, parent=None, args=None, macros=None):
...
+            statusLabel.setMaximumWidth(100)
+            statusLabel.setMaximumHeight(28)
+            statusLabel.setMinimumHeight(28)
             statusLabel.setSizePolicy(QSizePolicy.MinimumExpanding,
                                       QSizePolicy.MinimumExpanding)
+            statusLabel.setAlignment(Qt.AlignCenter)
+
+            # statusLabel.setFrameStyle(QFrame.Box | QFrame.Plain)
+
+            # statusLabel.setLineWidth(2)
```
Comment (Derikka) [inline cavityFaultDisplay.py]: If we don't need these commented out lines, maybe delete them?

Comment (lopardog) [reply cavityFaultDisplay.py]: fixed!

File: cavityFaultDisplay.py (hunk 1 above)
Comment (lopardog) [reply cavityFaultDisplay.py]: fixed! i commited and pushed again.

**Review by lisazacarias — APPROVED** [review]

**Comment by lisazacarias** [comment]
Excellent!

**Pull request merged**